lib/fastlane/plugin/aws_s3/actions/aws_s3_action.rb

```diff
@@ -89,7 +91,7 @@ def self.run(config)
         client_cfg[:region] = s3_region if s3_region
         client_cfg[:endpoint] = s3_endpoint if s3_endpoint
         client_cfg[:profile] = s3_profile if s3_profile
-        client_cfg[:credentials] = Aws::Credentials.new(s3_access_key, s3_secret_access_key) if s3_access_key && s3_secret_access_key
+        client_cfg[:credentials] = Aws::Credentials.new(s3_access_key, s3_secret_access_key, s3_session_token) if s3_access_key && s3_secret_access_key
```

Comment:
https://docs.aws.amazon.com/sdk-for-ruby/v2/api/Aws/Credentials.html

the default value of the session token filed is null, hence it should be okay